pom.xml

```diff
@@ -100,9 +102,12 @@
 		<codahale.metrics.version>3.0.0</codahale.metrics.version>
 		<avro.version>1.7.6</avro.version>
 		<jets3t.version>0.7.1</jets3t.version>
+                <commons.math3.version>3.2</commons.math3.version>
+                <commons.httpclient.version>4.3.6</commons.httpclient.version>
+                <io.netty.version>4.0.17.Final</io.netty.version>
```

Comment:
update please version of netty to latest version 4.0.25.Final
because spark 1.2.0+ use PooledByteBufAllocator with 8 parameters, which introduced in 4.0.18.Final
without update netty we have 
java.lang.NoSuchMethodError: io.netty.buffer.PooledByteBufAllocator.<init>(ZIIIIIII)V
in spark.version=1.2.0-cdh5.3.1when create SparkContext